src/sentry/snuba/metrics/datasource.py

```diff
@@ -73,6 +73,15 @@
 logger = logging.getLogger(__name__)
 
 
+def _build_use_case_id_filter(use_case_id: UseCaseID):
+    use_case_value = use_case_id.value
+    # For sessions, the `use_case_id` field in Clickhouse is stored as "".
+    if use_case_id == UseCaseID.SESSIONS:
+        use_case_value = ""
```

Comment:
Shouldn't we ask snuba to fix this ? 
If there is a use_case_id field in the table then it should be populated with `"session"` or something similar... don't know how easy (of if it is possible) to fix this but I think we should at least ask about it.

We may need for a while to search for both "" and "session" if we change snuba to set the `use_case_value` from now on.

Reply:
I have asked them, I might merge after I get an answer from them.

Reply:
I believe that fix is incoming. Either way, you will need to check for both `""` and `"sessions"` for the foreseeable future. I would add that check now, so that if/when the snuba change hits you don't have to change this again.

Reply:
Perfect, will do!

The thing though, is that I think that such endpoint(s) should not know about the concept of use case id. This applies to all the metrics endpoints.